test/unit/ai-service.test.ts

```diff
@@ -116,10 +116,42 @@ describe('AI-service', () => {
       const aiRewriteContext = DUMMY_AI_REWRITE_CONTEXT;
 
       mockFetch.mock(getGetAIChatCompletionMatcher(count, internalName), {
-        status: 401,
-        throws: {
-          code: 401,
-          message: 'Unauthorized',
+        status: 500,
+        body: {
```

Comment:
is AI service error mapped correctly? 
it has httpsErrorCode, errorId & errorDescription

Reply:
seems ai-service is encapsulating its error response.